src/librustc_middle/mir/interpret/allocation.rs

```diff
@@ -544,18 +544,18 @@ impl<'tcx, Tag: Copy, Extra> Allocation<Tag, Extra> {
 
 /// Undefined bytes.
 impl<'tcx, Tag: Copy, Extra> Allocation<Tag, Extra> {
-    /// Checks whether the given range  is entirely defined.
+    /// Checks whether the given range  is entirely initialized.
     ///
-    /// Returns `Ok(())` if it's defined. Otherwise returns the range of byte
+    /// Returns `Ok(())` if it's initialized. Otherwise returns the range of byte
     /// indexes of the first contiguous undefined access.
```

Comment:
Another remaining "undef".

Reply:
I searched again through allocation.rs and found this comment as well as one other and a variable, which were all changed appropriately. I left comments which were for other functions / parts of the code which have yet to be changed (eg. `inspect_with_undef_and_ptr_outside_interpreter`).